html/css/theme.css

```diff
@@ -53,7 +53,7 @@ h1 {
 }
 
 .home-box {
-	height: 350px;
```

Comment:
why?

Reply:
This is a shortsighted design I made for the original emoji server, where the search bar was positioned absolutely relative to the bottom of the screen. Otherwise, the intro text and the search bar would overlap. I would like to fix this layout issue in the future, because I don't like the fact that I hard coded a height either.